Source/WebKit/UIProcess/API/glib/WebKitWebView.cpp

```diff
@@ -5332,3 +5337,11 @@ webkit_web_view_get_default_content_security_policy(WebKitWebView* webView)
 
     return webView->priv->defaultContentSecurityPolicy.data();
 }
+
+void webkitWebViewRestartWebProcessResponsivenessTimerForTesting(WebKitWebView* webView)
+{
+    g_return_if_fail(WEBKIT_IS_WEB_VIEW(webView));
```

Comment:
Since this is not public API, no need for soft asserts:

```suggestion
    ASSERT(WEBKIT_IS_WEB_VIEW(webView));
```